WBDynamics/va-website-template
# Feature 5943877783 tinypng key and dsstore file removal

## What type of PR is this? (check all applicable)


## Description
    Added repo secret for the compress workflow.
    removed the .DS_Store file 
    Added .DS_Store to the .gitignore file in any directory

## Related Tickets & Documents
- Monday Task #5943877783

## QA Instructions, Screenshots, Recordings

-see exclude for ds_store file in gitignore
-see tinypng api key added to compress.yml

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_

## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized

## Changed Files in this Pull Request ##
- .github/workflows/compress.yml
 - .gitignore
